Welolo/cypress/integration/recipient_form.js

```diff
@@ -0,0 +1,37 @@
+import InputFields from './InputField.js';
+const inputFields = new InputFields();
+
+describe('Verify input fields are present', () => {
+    beforeEach(()=>{
+        cy.visit('http://localhost:3000')
+    })
+    it('Visits the site & verifies elements', () => {
+        inputFields.reciepientName().should('be.visible')
+        inputFields.reciepientPhone().should('be.visible')
+        inputFields.fundQuantity().should('be.visible')
+        inputFields.message().should('be.visible')
+    })
+    it('verify the correct input!', () => {
```

Comment:
Maybe some tests with invalid information (like 208123456u) as the phone number or random symbols just placed in fields that don't allow them might be helpful to have since we want to validate those. Otherwise I really like what you have so far!

Reply:
Yes, for validating the inputs I am working on other tests outside Cypress. Task #31 is validating the user inputs. Is it necessary to check that here as well?

Reply:
It might be helpful to have to ensure we did the validation right. For example, if we test the phone number and for some reason it actually gets accepted with no error message, we know there's something in the validation is wrong.